### ISSUE TITLE
v1.7 RC Basic Auth KV and Web UI

### ISSUE THREAD

**Issue by michaelpporter** [containous/traefik#3612]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

The old format for KV Basic Auth was

`traefik/frontends/frontend1/basicauth/0 'user:password'`

The new value is

`traefik/frontends/frontend1/auth/basic/users/0 'user:password'`

When I remove the old format and replace it with the new I do not get prompted for a password.
Traefik Dashboard does not show either format in the frontend details tab.

### What did you expect to see?

I expected to see a password for the frontend and a value displayed on the details tab

### What did you see instead?

Nothing.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
traefik version
Version:      v1.7.0-rc1
Codename:     maroilles
Go version:   go1.10.3
Built:        2018-07-09_03:03:16PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

Using Consul on Ubuntu 16.04 

```toml
################################################################
# Global configuration
################################################################

# Enable debug mode
debug = true
logLevel = "WARN"

# EntryPoints are the ports that are open for Træfik to respond to.
defaultEntryPoints = ["http", "https"]
[entryPoints]
  [entryPoints.api]
    address = ":8081"
  [entryPoints.http]
  address = ":80"
    [entryPoints.http.redirect]
      entryPoint = "https"
  [entryPoints.https]
  address = ":443"
    [entryPoints.https.tls]

# Acme setting are used for letsencrypt to get the wildcard cert.
[acme]
email = "admin@example.com"
storage = "traefik/acme/account"
caServer = "https://acme-v02.api.letsencrypt.org/directory"
entryPoint = "https"
  [acme.dnsChallenge]
  provider = "digitalocean" # DNS Provider name (cloudflare, OVH, gandi...)
  delayBeforeCheck = 0
[[acme.domains]]
  main = "*.example.com"

# Consul entry point settings, where Træfik looks for the KV store.
[consul]
  endpoint = "127.0.0.1:8500"
  watch = true
  prefix = "traefik"

[traefikLog]
  filePath = "/var/log/traefik/traefik.log"
[accessLog]
  filePath = "/var/log/traefik/access.log"

[api]
  entrypoint = "api"
```

```bash
# New
SITE='php7a'
consul kv put traefik/frontends/$SITE/backend php7a
consul kv put traefik/frontends/$SITE/entrypoints http,https
consul kv put traefik/frontends/$SITE/routes/route0/rule Host:$SITE.example.com
consul kv put traefik/frontends/$SITE/priority 100
consul kv put traefik/frontends/$SITE/passhostheader true
consul kv put traefik/frontends/$SITE/auth/basic/users/0 'user:password'

# Old
SITE='php7a'
consul kv put traefik/frontends/$SITE/backend php7a
consul kv put traefik/frontends/$SITE/entrypoints http,https
consul kv put traefik/frontends/$SITE/routes/route0/rule Host:$SITE.example.com
consul kv put traefik/frontends/$SITE/priority 100
consul kv put traefik/frontends/$SITE/passhostheader true
consul kv put traefik/frontends/$SITE/basicauth/0 'user:password'
```

### If applicable, please paste the log output in DEBUG level (`--logLevel=DEBUG` switch)

```
time="2018-07-12T12:57:37-05:00" level=warning msg="Deprecated configuration found: /basicauth. Please use /auth/basic/.
```

Edited to remove real domains

**Comment by michaelpporter** [comment]
I might be reading this wrong...I do not know the value of rootPath in these checks.

It seems to me the function [hasDeprecatedBasicAuth](https://github.com/containous/traefik/blob/c7e008f57a143a02cb2899c8c98859422169031c/provider/kv/kv_config.go#L373) is checking the whole store, not just the active frontend.

If so then [basicAuth.Users = p.getList(rootPath, pathFrontendAuthBasicUsers)](https://github.com/containous/traefik/blob/c7e008f57a143a02cb2899c8c98859422169031c/provider/kv/kv_config.go#L408) is never called and all frontends would need to be updated to prevent this.

Does it make since to check for the new format first, then fall back to the deprecated configuration?

**Comment by ldez** [comment]
the function ` hasDeprecatedBasicAuth` use a variable name `rootPath`.
This `rootPath` is the path to one frontend not to all the store.

**Comment by traefiker** [comment]
Closed by #3615.

**Comment by traefiker** [comment]
Closed by #3628.
